src/thor/route_action.cc

```diff
@@ -363,7 +380,7 @@ void thor_worker_t::path_arrive_by(Api& api, const std::string& costing) {
         get_path_algorithm(costing, *origin, *destination, api.options());
     path_algorithm->Clear();
     algorithms.push_back(path_algorithm->name());
-    valhalla::midgard::logging::Log(path_algorithm->name(), " [ANALYTICS] algorithm::");
+    LOG_INFO(std::string("algorithm::") + path_algorithm->name());
```

Comment:
for now i left this in as a LOG_INFO since its the biggest performance differentiator. i could be convinced to remove it though, thoughts?

Reply:
I find it helpful when debugging issues to at least as a sanity check that the right a-star is handling the request. I'd vote for keeping it.

Reply:
yeah, +1 on keeping it.